include/rocksdb/uuid.h

```diff
@@ -0,0 +1,117 @@
+//  Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+
+#pragma once
+
+#include <cstdint>
+
+#include "rocksdb/status.h"
+
+namespace ROCKSDB_NAMESPACE {
+
+// A base struct / holder for an identifier of up to 128 bits. All zeros is
+// considered empty, so implementations should avoid generating / saving the
+// empty value when a value is actually generated.
+struct RawUuid {
+  //== Data
+  uint64_t lower64 = 0;
+  uint64_t upper64 = 0;
+
+  //== Construction
+  RawUuid() { assert(IsEmpty()); }
+
+  static RawUuid FromRawBytes(const char *buf_of_16);
```

Comment:
Why is this not the same signature as RfcUuid::Parse?  It seems like they should be equivalent.